ee/clickhouse/sql/funnels/funnel.py

```diff
@@ -1,5 +1,5 @@
 FUNNEL_PERSONS_BY_STEP_SQL = """
-SELECT person_id
+SELECT person_id {timestamp}
```

Comment:
maybe this should be `extra_fields` ?

Can also wait till we actually have other fields, like when we get to path pinning.